security/security-spring/src/main/java/org/cbioportal/security/spring/authentication/keycloak/SAMLUserDetailsServiceImpl.java

```diff
@@ -107,8 +107,7 @@ public Object loadUserBySAML(SAMLCredential credential)
             if (attrName.equals(SAML_IDP_METADATA_ROLE_ATTR_NAME)) {
                 List<XMLObject> attributeValues = cAttribute.getAttributeValues();
                 if (!attributeValues.isEmpty()) {
-                    userRoles.add(new StringBuilder(APP_NAME).append(":").append(
-                        getAttributeValue(attributeValues.get(0))).toString());
+                    userRoles.add(getAttributeValue(attributeValues.get(0)));
```

Comment:
@pieterlukasse @fcriscuo ,we will have to use "**APP_NAME**" to support both models (authorization by keycloak or other identity providers) till we stop using **users** and **authorities** two database tables.